test/scenarios.py

```diff
@@ -7227,6 +7227,174 @@
             ),
         },
     ),
+    Scenario(
+        "scenario29-14",
+        "Send a bunch of files and wait until some of them arrive. Then stop the sender and cancel the transfer on receiver. Expect the file states match on both sides",
```

Comment:
The description lacks the part where the seender must come online in order to sync

Reply:
Done